toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -185,7 +204,8 @@ func outputArtifacts(ctx context.Context, items []imagecustomizerapi.OutputArtif
 
 	err = writeInjectFilesYaml(outputArtifactsMetadata, outputDir)
 	if err != nil {
-		return fmt.Errorf("failed to write inject-files.yaml:\n%w", err)
+		return fmt.Errorf("%w: %w", ErrArtifactYamlWrite,
+			fmt.Errorf("failed to write inject-files.yaml:\n%w", err))
```

Comment:
All the errors in the file should be using an error object without a nested fmt.Errorf(), fix these any others as I've mentioned before